README.rst

```diff
@@ -70,3 +70,41 @@ To generate correct files, please provide the following input to Cookiecutter:
                                  super project of this module, e.g. Invenio
                                  itself.
 ================================ =============================================
+
+Usage example
+--------------
```

Comment:
I was creating a section called 'Usage example' at the same level as 'configuration', with the others as subsections (requirements, create a new python module...).

It makes no sense to me to put them all at the same level.